bin/crew

```diff
@@ -249,8 +249,28 @@ end
 def update
   abort "'crew update' is used to update crew itself. Use 'crew upgrade <package1> [<package2> ...]' to update specific packages.".orange if @pkg_name
 
-  # update package lists
+  # update crew from git.
   Dir.chdir(CREW_LIB_PATH) do
+    # Fix git checkout if .git is missing.
+    unless Dir.exist?(File.join(CREW_LIB_PATH, '.git'))
+      puts 'Fixing Chromebrew system git repo clone...'.orange
+      # Run git setup commands from install.sh
+      Dir.chdir(CREW_LIB_PATH) do
+        # Make the git default branch error messages go away.
+        `git config --global init.defaultBranch main`
+        # Setup the dir with git information.
+        `git init --ref-format=reftable`
+        `git remote add origin #{CREW_REPO}`
+        # Help handle situations where GitHub is down.
+        `git config --local http.lowSpeedLimit 1000`
+        `git config --local http.lowSpeedTime 5`
+        # Checkout, overwriting local files.
+        system 'git fetch --all', %i[out err] => File::NULL
+        system 'git checkout -f master', %i[out err] => File::NULL
```

Comment:
Why use `system` instead of backticks?

Reply:
So I can use `%i[out err] => File::NULL`.

Reply:
Would it be possible (and maybe even faster) if all the git commands were executed in a single system call?

Reply:
What would be the readability expense?  Maybe the commands could be grouped together in a doc block?

Reply:
Done.